src/GI.h

```diff
@@ -9,6 +9,7 @@ template <class Ptcl> class GI : public Problem<Ptcl>{
     public:
 	static double end_time;
     static double damping;
+    static std::string output_directory;
```

Comment:
I do not think this is the right place for that variable to live in. It is not related to the initial condition, it is more a global variable for the model. If we put it in the right place it also does not have to be global anymore. I will indicate further down where to put it.

Reply:
You still need to remove this variable